src/tests/detect-template-buffer.c

```diff
@@ -49,7 +49,7 @@ static int DetectTemplateBufferTest(void)
     p->flow = &f;
     p->flags |= PKT_HAS_FLOW | PKT_STREAM_EST;
     p->flowflags |= FLOW_PKT_TOSERVER | FLOW_PKT_ESTABLISHED;
-    StreamTcpInitConfig(TRUE);
+    StreamTcpInitConfig(true);
```

Comment:
also not executed, wonder if that is a bug

Reply:
`DetectTemplateBufferRegisterTests` isn't invoked ... I'll update this PR with a fix.

Reply:
The tests are never registered because of this code in `DetectTemplateBufferRegister`
```
     if (ConfGetNode("app-layer.protocols.template") == NULL) {
          return;
     }
 ```